src/server/saved_objects/service/lib/repository.js

```diff
@@ -36,6 +36,7 @@ export class SavedObjectsRepository {
       schema,
```

Comment:
I'm not seeing us handling hidden types for the `bulkCreate`,  or `incrementCounter`. Was this an omission or on purpose?

Reply:
It was an omission on my part. Don't know what I was thinking. The checks are there now.